README.md

```diff
@@ -202,18 +201,39 @@ $ cargo run --bin hello --force-backtrace
 
 ## Troubleshooting
 
-### `probe-run --list-probes` says "No devices were found."
+### "Error: no probe was found."
 
-Apart from a faulty connection between your computer and the target device, this could be caused by several things:
+First, check your data cable:
+
+- make sure that it is connected to the right port on your development board
+- make sure that you are using a **data** cable– some cables are built for charging only! When in doubt, try using a different cable.
+
+If this doesn't resolve the issue, try the following:
 
 #### [Linux only] udev rules haven't been set
 
-In order for `probe-run` to find the device you'd like to run your code on, your system needs permission to access the device as a non-root user.
+Check if your device shows up in `lsusb`:
+
+```console
+$ lsusb
+Bus 001 Device 008: ID 1366:1015 SEGGER J-Link
+# if your device shows up like this ^^^^^^^^, skip to the next troubleshooting section
```

Comment:
What are common debuggers, except "J-link"? Maybe we can give them a few keywords to scan for, in case they have multiple USB devices and don't know if one is their debugger or not.

Reply:
afaik J-Links are by far the most common. I'd rather not add lots of text by trying to create a comprehensive list (in my experience, the longer the tutorial the more likely it won't be thoroughly read and the info you're putting in there is lost in most cases)
If you're unsure, you can always unplug, `lsusb`, plug in, `lsusb`, and look at the difference. (I could add that as a tip?)

Reply:
Yes, I think this is a useful tip!